src/Microsoft.AspNetCore.Authentication/AuthenticationHandler.cs

```diff
@@ -233,12 +233,28 @@ public bool ShouldHandleScheme(string authenticationScheme, bool handleAutomatic
             }
         }
 
+        /// <summary>
+        /// Handle the authentication. If a task alreay exists to handle the authentication, return the task.
+        ///
+        /// Exception is thrown from the HandleAuthenticateAsync task will be handled and saved to AuthenticateResult.
+        /// </summary>
         protected Task<AuthenticateResult> HandleAuthenticateOnceAsync()
         {
             if (_authenticateTask == null)
             {
-                _authenticateTask = HandleAuthenticateAsync();
+                try
+                {
+                    _authenticateTask = HandleAuthenticateAsync().ContinueWith<AuthenticateResult>(
```

Comment:
Given that we don't consistently expect/check errors from this, we should probably Log any failures from this method